client/my-sites/stats/site.jsx

```diff
@@ -522,6 +531,37 @@ function StatsBody( { siteId, chartTab = 'views', date, context, isInternal, ...
 	// TODO: Fix isOdysseyStats to include the environment running on WP-Admin of Simple sites.
 	const isRunningOnWPAdmin = document.getElementById( 'wpadminbar' );
 
+	const statsAdminVersion = useSelector( ( state ) =>
```

Comment:
Should we move these to `PageModuleToggler` as well?

Reply:
Yup, I was planning to push it in this PR. Thank you for merging this, though, as this at least merges some part of code and not all is blocked. I will handle this in a separate PR